playbooks/gcp/openshift-cluster/filter_plugins

```diff
@@ -0,0 +1 @@
+../../../filter_plugins
```

Comment:
We are no longer doing filter_plugins this way.  All filter plugins should be moved to lib_utils and that role should be brought into scope via a meta-depends in a role or at the play level in the 'roles:' section.

Reply:
I'm not using any special filter plugin, I'll run this without.